### PR TITLE
Add Finnish imagery

### PR DESCRIPTION
**Pull request by NKAmapper**
Here are 5 maps for Finland:

**From MML - The National Land Survey of Finland**
- Topo map
- Background map
- Ortophotos (best imagery for Finland)

[Explicit permission from MML](https://forum.openstreetmap.org/viewtopic.php?pid=356296#p356296)

**From FTA - The Finnish Transport Agency**
- Digiroad Road network - functional road classes (overlay)
- Digiroad Road network - endpoint wms with additional features (overlay)

[Explicit permission from FTA](https://www.facebook.com/download/preview/1324879040926767)

[Imagery in JOSM](https://josm.openstreetmap.de/wiki/Maps/Finland)

#337

### CODE REVIEW

File: sources/europe/fi/fta-digiroad-functional.geojson [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        "icon": "https://www.liikennevirasto.fi/livi-theme/images/general/liikennevirasto_logo_2x.png",
+        "min_zoom": 3,
+        "max_zoom": 20,
+        "overlay": true,
+        "attribution": {
+            "required": true,
+            "text": "\u00a9 Liikennevirasto",
+            "url": "https://www.liikennevirasto.fi/web/en/open-data/digiroad/"
+        },
+        "license_url": "https://lookaside.fbsbx.com/file/Digiroad_permission_23032017.pdf?token=AWzxeAA6eoURtiEfhamunEmttk0mdOG8xdWL0xQJ2ycs6k0AB2M3DQVY11G1Cg8VQpkoW8PoiPfXFrocjWpmietbZi5EtBGPk-coh5sgCxeArDFwVSmmEaCRccgzstqIN4bA5PYnXJSISjW-K8AnAcaw",
```
Comment (tyrasd) [inline sources/europe/fi/fta-digiroad-functional.geojson]: I cannot access this pdf document. Am I doing something wrong?

Comment (NKAmapper) [reply sources/europe/fi/fta-digiroad-functional.geojson]: Apologies, here is the correct link: https://www.facebook.com/download/preview/1324879040926767

(You will need to be logged into a Facebook account first)

**Comment by grischard** [comment]
Thank you for this PR! The boundaries seem a bit rough, which could annoy your neighbours. Would it please be possible to use the downloads.geofabrik.de poly?

**Comment by Klumbumbus** [comment]
The more precise boundary could be used on the other two MML an the two FTA sources too.

**Comment by theel0ja** [comment]
The copyright text "Maanmittauslaitoksen" (genitive) should be changed to "Maanmittauslaitos".

**Comment by NKAmapper** [comment]
#479

**Comment by theel0ja** [comment]
HTTPS should be enabled, as `https://tiles.kartat.kapsi.fi/` seems to support it.

**Comment by grischard** [comment]
I've added https.

**Pull request merged**